### PR TITLE
Forte: Add service_fee_amount field

### PR DESCRIPTION
**Pull request by meagabeth**
Add support for service fee which can be passed in the request body as `service_fee_amount`. To note, merchants must be configured to accept service/convenience fees or they will get a 'U27: CONV FEE NOT ALLOWED' [transaction response code](https://www.forte.net/devdocs/reference/response_codes.htm). The sandbox environment I am working with is not configured for service fees so I was unable to write a remote test for this.

CE-491

Unit:
21 tests, 72 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
Remote:
22 tests, 63 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: test/unit/gateways/forte_test.rb [hunk 1]
```diff
@@ -61,6 +61,16 @@ def test_failed_purchase_with_echeck
     assert_equal 'INVALID CREDIT CARD NUMBER', response.message
   end
 
+  def test_successful_purchase_with_service_fee
+    response = stub_comms(@gateway, :raw_ssl_request) do
+      @gateway.purchase(@amount, @credit_card, @options)
+    end.respond_with(MockedResponse.new(successful_purchase_response))
```
Comment (therufs) [inline test/unit/gateways/forte_test.rb:67]: How would we feel about adding a `successful_purchase_response_with_service_fee`, which we could crib from https://restdocs.forte.net/?version=latest#c8c05c2d-b18e-466a-bcf0-178a4e165895?

Comment (meagabeth) [reply test/unit/gateways/forte_test.rb]: And done!

**Review by naashton — APPROVED** [review]
Looks good! 👍

**Comment by meagabeth** [comment]
This was merged successfully - closing manually.

**Pull request closed without merging**